packages/generator-electric/generators/app/templates/src/partials/Footer.soy

```diff
@@ -0,0 +1,19 @@
+{namespace Footer}
+/**
```

Comment:
Add line break and update comment block to use the same pattern we use on other files